translations/en.yml

```diff
@@ -327,45 +354,22 @@ src:
               projects, assets and datasets for all team members.
             '3399326760': I am sure I want to delete this workspace.
             '4086722238': Please type your workspace name to continue.
-          Dataset:
-            DatasetItem:
-              '2065124393': Delete this dataset
-              '3112669357': Are you sure you want to delete it?
-              '3528672691': Cancel
-              '3645414921': Delete
-            DatasetSection:
-              '293739158': Add Dataset
-              '3833270271': Dataset
           MemberListItem:
             '1068157859': Writer
             '1954432048': Reader
             '3470351671': Owner
           MembersSection:
             '721495104': New member
             '2734377620': Members
-          Plugin:
-            PluginAccordion:
-              PluginAccordionItem:
-                '31593394': Uninstall
-                '730907263': Installed
-                '3480550112': Install
-            PluginSection:
-              '3060204202': Under Construction. Coming Soon.
-              PluginInstall:
-                '149186996': Uploaded plugin
-                '311550959': Private GitHub repository
-                '1361263844': Public GitHub repository
-                '1889877734': Plugins lists
-                '3129492077': Zip file from PC
-              PluginList:
-                '1933254449': Advanced options
           ProfileSection:
             '2999985900': Workspace name
         WorkspaceList:
           WorkspaceCell:
-            '293600023': 'Team Members:'
-            '2968463849': This is your personal account
-            '3038786653': No Title Project
+            '808766300': No Title Workspace
+            '974692228': >-
+              This is your personal workspace, your projects and resource will
+              be managed in this workspace.
```

Comment:
This is your personal workspace. Your projects and resources will be managed in this workspace.